projects/igniteui-angular/src/lib/tabs/tabs.component.spec.ts

```diff
@@ -397,6 +412,65 @@ describe('IgxTabs', () => {
         const indicator = dom.query(By.css('.igx-tabs__header-menu-item-indicator'));
         expect(indicator.nativeElement.style.width).toBe('90px');
     }));
+
+    it('should navigate to the correct URL when clicking on tab buttons', fakeAsync(() => {
```

Comment:
Can we add a test where some tab is initially selected?

Reply:
It is no possible to have an initially selected tab in this component's mode. Both tab controls cannot navigate by them self's to a specific URL. This can be done in the application's logic by raising a click event on the tab item's native elements but we have such test.

Reply:
After discussion, this will be a test for the `isSelected` input, not related to navigation.